packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.js

```diff
@@ -0,0 +1,230 @@
+import SfIcon from "@/components/atoms/SfIcon/SfIcon.vue";
+import SfLoader from "@/components/atoms/SfLoader/SfLoader.vue";
+import SfStore from "./_internal/SfStore.vue";
+
+Vue.component("SfStore", SfStore);
+
+import Vue from "vue";
```

Comment:
shouldn't this import be at the top? Not sure if it breaks anything but linter is yelling ;)

Reply:
Yes of course it should :D Sorry!